client/components/deposits-overview/index.tsx

```diff
@@ -119,6 +193,12 @@ const DepositsOverview: React.FC = () => {
 						depositsSchedule={ account.deposits_schedule }
 						showNextDepositDate={ availableFunds > 0 }
 					/>
+					<ClickTooltip
+						content={ nextDepositHelpContent }
+						className="wcpay-deposits-overview__schedule__tooltip"
+						buttonIcon={ <HelpOutlineIcon /> }
+						buttonLabel={ 'Deposit schedule tooltip' }
+					/>
```

Comment:
⛏️ this `className` may be redundant, AFAICT it isn't references anywhere.

Reply:
Removed! adfc76dee01a5b9f57937de92e5bf0b38fe1191a